docs/src/osdk/reference/environment-variables.md

```diff
@@ -0,0 +1,18 @@
+# Environment Variables
+
+The OSDK tool uses the following environment variables to customize compilation and target behavior. 
+
+## OSDK-Specific Variables
+
+### `INTEL_TDX`
```

Comment:
Remove this one. This environment variable is only used by an internal function for test cases. So it should not be documented publicly.

https://github.com/asterinas/asterinas/blob/1452aab69c9d0bf5ada7ea8a93a49fd0badafcca/osdk/src/commands/util.rs#L29-L31